frame/support/src/storage/bounded_vec.rs

```diff
@@ -319,6 +319,21 @@ impl<
 	}
 }
 
+impl<T, S> BoundedEncodedLen for BoundedVec<T, S>
+where
+	T: BoundedVecValue + BoundedEncodedLen,
+	S: Get<u32>,
+	BoundedVec<T, S>: Encode,
+{
+	fn max_encoded_len() -> usize {
```

Comment:
the least significant comment: pity not to see `SomeTrait { fn some_trait() }`, why not `trait MaxEncodedLen`?

Reply:
No reason really; I like `MaxEncodedLen` because it's shorter and agrees with the trait's function. https://github.com/paritytech/substrate/pull/8720/commits/af0b8146e20b319985b758c38c6fe33451d51c7a